ludwig/hyperopt/execution.py

```diff
@@ -146,7 +146,7 @@ def __init__(
         metric: str,
         goal: str,
         split: str,
-        search_alg: Dict,
+        search_alg: "BaseSearchAlgorithmConfig",  # noqa: F821
```

Comment:
nit: Can we maybe import at the top? Should help get rid of the noqa

Reply:
Oh, whoops, that should actually be `Dict`, made that change expecting to receive a schema object here. On a separate note, I think it would be worthwhile to load the hyperopt schema from dict in `hyperopt/run.py` and use the OO config throughout. That could be a low-priority followup PR.